app/views/posts/index.html.erb

```diff
@@ -29,12 +29,15 @@
           </div>
           <h4 class="comment-title">Comments</h4>
           <div class="comment-container">
-            <% post.recent_comments.each do |comment|%>
+            <% post.comments.each do |comment|%>
```

Comment:
- [ ] **[OPTIONAL]** If I recall correctly, from a previous project, the post index view should render the recent comments, not all comments. Kindly revert to the previous code you have.